### PR TITLE
fix: Fix player list not rendering when overlay is disabled

### PR DESCRIPTION
**Pull request by kristofbolyai**

### CODE REVIEW

File: common/src/main/java/com/wynntils/features/overlays/CustomPlayerListFeature.java [hunk 1]
```diff
@@ -46,7 +47,8 @@ public class CustomPlayerListFeature extends Feature {
 
     @SubscribeEvent
     public void onRender(RenderEvent.Pre event) {
-        if (event.getType() == RenderEvent.ElementType.PLAYER_TAB_LIST) {
+        if (event.getType() == RenderEvent.ElementType.PLAYER_TAB_LIST
```
Comment (magicus) [inline common/src/main/java/com/wynntils/features/overlays/CustomPlayerListFeature.java:50]: That's interesting. Why do disabled features get events? 🤔 

That sounds like a bug to me. I think the feature itself should be able to rely on that it is enabled when receiving an event.

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/overlays/CustomPlayerListFeature.java:50]: If you only disable the overlay...

Comment (magicus) [reply common/src/main/java/com/wynntils/features/overlays/CustomPlayerListFeature.java:50]: Ah, I see... Ok, then this is fine.

Comment (magicus) [reply common/src/main/java/com/wynntils/features/overlays/CustomPlayerListFeature.java:50]: (But maybe we should do something about the way we end up with "overlay-wrapping features"; somehow promote the overlay themselves to proper features?)

**Pull request merged**